tests/basic/build_date.phpt

```diff
@@ -0,0 +1,8 @@
+--TEST--
+PHP_BUILD_DATE
+--FILE--
+<?php
+var_dump(PHP_BUILD_DATE);
+?>
+--EXPECTREGEX--
+string\(20\) "[A-Za-z]{3} \d{2} \d{4} \d{2}:\d{2}:\d{2}"
```

Comment:
I see that this is merged already, but I think `__DATE__` format has a quirk of space-padded date numbers on single-digit dates like `Sep  4 2024`. 

Perhaps something like this?

```diff
- string\(20\) "[A-Za-z]{3} \d{2} \d{4} \d{2}:\d{2}:\d{2}"
+ string\(20\) "[A-Za-z]{3} ( \d|\d{2}) \d{4} \d{2}:\d{2}:\d{2}"
```